aten/src/ATen/CMakeLists.txt

```diff
@@ -415,6 +424,11 @@ if(NOT MSVC AND NOT EMSCRIPTEN AND NOT INTERN_BUILD_MOBILE)
 endif()
 
 if(USE_CUDA AND NOT USE_ROCM)
+  # add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/../../../third_party/cutlass" ${CMAKE_BINARY_DIR}/cutlass)
+  # list(APPEND ATen_THIRD_PARTY_INCLUDE ${TORCH_ROOT}/third_party/cutlass/include)
+  if(CMAKE_CUDA_COMPILER_VERSION VERSION_GREATER_EQUAL 11.6)
+    list(APPEND ATen_CUDA_INCLUDE ${CMAKE_CURRENT_SOURCE_DIR}/../../../third_party/cutlass/include)
```

Comment:
This condition should match flash_attn condition

Reply:
Correct me if I'm wrong, but this would make cutlass a requirement for all users of libtorch. It's better to cmake this a private include dependency of `libtorch_cuda`

Reply:
@malfet Internal seems to be appeased by the ifdef changes, otherwise a code pointer for this change would be great and I think its ready to land